purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1046,6 +1046,22 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         )
     }
 
+    /**
+     * Subscriber attribute associated with the Airship Channel ID
+     * Required for the RevenueCat Airship integration
+     *
+     * @param airshipChannelID null will delete the subscriber attribute
```

Comment:
do we want to do any special handling for empty values?

Reply:
we already have it, although I'm now realizing that the docstring wouldn't lead you to expect it
https://docs.revenuecat.com/docs/subscriber-attributes#deleting-attributes
I'll update